wordpress-mobile/WordPress-Android
# Blogging Reminders: Replace references to "goals" with "reminders"

Fixes #14895

This updates and renames various strings in order to replace all references to "goals" with "reminders". More specifically:

- Settings screen:
  - Blogging ~goals~ reminders
- First screen (Prologue):
  - Title: Set your blogging ~goals~ reminders
  - Body: Your post is publishing… in the meantime, set up your blogging ~goals to get reminders, and track your progress.~ reminders on days you want to post.
  - Button: Set ~goals~ reminders
- Last screen (Epilogue):
  - You can update this anytime via My Site > Settings > Blogging ~goals.~ reminders.

To test:

- Make sure you have built a debug variant of the app.
- Clear app data and log in.
- Make sure the feature flag is turned on.

Go through the entire Blogging Reminders flow and make sure no reference to "goal" exists.

## Regression Notes
1. Potential unintended areas of impact
None.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests.

PR submission checklist:
